### ISSUE TITLE
Behaviour of Picture Canvas

### ISSUE THREAD

**Issue by meurigbeynon** [EMGroup/js-eden#178]
In the initial Canvas Picture display of the CONSTRUIT! environment, the logo is defined to be 'in the middle of the canvas'. If you expand the canvas to change its size by dragging, the logo image moves accordingly. If you use the Full Screen button to expand the canvas window the logo doesn't move to the middle of the screen. Does this represent a bug?

**Comment by ElizabethHudnott** [comment]
Yes, it represents a bug, at least kind of a bug, depending which way you look at.  It depends on whether _view_xxx_width and _view_xxx_height are recording the size of the window as a specification of what the window's size is defined to be, or whether they are supposed to record the actual physical size of window as it is now (while maximized, which is only temporary, before the physical size reverts to the "specified size" when the window is unmaximized).
